src/lesson8/Task2.java

```diff
@@ -0,0 +1,32 @@
+package lesson8;
+
+//        2. Fix the method with try-catch to get the correct result of division:
+//
+//public class Main {
+//
+//    public static void main(String[] args) {
+//        System.out.println(divideByZero());
+//    }
+//
+//    public static int divideByZero() {
+//        int a = 5;
+//        int b = 0;
+//        return a / b;
+//    }
+//}
+public class Task2 {
+    public static void main(String[] args) {
+        try {
+            divideByZero();
+        } catch (ArithmeticException e) {
+            System.out.println(e.getMessage());
+
+        }
+    }
+
+        public static int divideByZero() {
+            int a = 5;
+            int b = 0;
+            return a / b;
+        }
```

Comment:
Try/catch should be implemented inside divideByZero() method otherwise you have to handle exception everywhere where you invoke this method.